be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -48,11 +48,10 @@ ColumnChunkReader::ColumnChunkReader(level_t max_def_level, level_t max_rep_leve
 ColumnChunkReader::~ColumnChunkReader() = default;
 
 Status ColumnChunkReader::init(int chunk_size) {
-    int64_t start_offset = 0;
-    if (metadata().__isset.dictionary_page_offset) {
+    int64_t start_offset = metadata().data_page_offset;
```

Comment:
could you extract this code to a function `has_dict_page`? and rewrite following code to 

```
if (has_dict_page(metadata)) {
   start_offset = metadata.dictionary_page_offset;
}
```

Reply:
+1 and please add some comments